nixos/modules/system/boot/loader/systemd-boot/systemd-boot.nix

```diff
@@ -351,6 +353,21 @@ in {
     };
   };
 
+    defaultEntry = mkOption {
+      default = null;
+
+      type = types.nullOr types.str;
+
+      description = ''
+        Default boot entry. Sets the "default" configuration option
+        of the loader.conf file.
+
+        If not set, the latest Nix generation will be used as the default entry.
+
+        See <https://www.freedesktop.org/software/systemd/man/latest/loader.conf.html#Options>.
+      '';
+    };
```

Comment:
This option is outside of the scope of `options.boot.loader...`.

Reply:
I see it. Thanks.